docker-storage-setup.spec

```diff
@@ -54,9 +38,12 @@ install -p -m 755 %{SOURCE5} %{buildroot}/%{dsslibdir}/dss-child-read-write
 %files
 %{_unitdir}/docker-storage-setup.service
 %{_bindir}/docker-storage-setup
-%{dsslibdir}/docker-storage-setup
```

Comment:
Why are we removing above. We put default config options in /usr/lib/docker-storage-setup/ and want to overwrite it on all package upgrades.